lldb/test/API/lang/swift/async/stepping/step-in/task-switch/TestSwiftTaskSwitch.py

```diff
@@ -15,11 +15,25 @@ def test(self):
         target, _, thread, _ = lldbutil.run_to_source_breakpoint(self, "await f()", src)
         self.assertEqual(thread.frame[0].function.mangled, "$s1a5entryO4mainyyYaFZ")
 
-        function = target.FindFunctions("$s1a5entryO4mainyyYaFZTQ0_")[0].function
-        instructions = list(function.GetInstructions(target))
+        sym_ctx_list = target.FindFunctions("$s1a5entryO4mainyyYaFZTQ0_", lldb.eFunctionNameTypeAuto)
+        self.assertEquals(sym_ctx_list.GetSize(), 1)
+        symbolContext = sym_ctx_list.GetContextAtIndex(0)
+        function = symbolContext.GetFunction()
+        if function:
+            instructions = list(function.GetInstructions(target))
+        else:
+            symbol = symbolContext.GetSymbol()
+            self.assertIsNotNone(symbol)
+            instructions = list(symbol.GetInstructions(target))
```

Comment:
this seems like the crux of the change, is that correct? Does this mean the test is being run without debug info? Or does it mean that even with debug info, somehow `GetFunction` returns `None`? If the reason for this `else` case is understood, then could you add a comment explaining?

Reply:
Yes.

Reply:
I may know what the problem is. If I understand correctly, `GetFunction` is taken from the debuginfo.
An incorrect `DW_TAG_subprogram` caused the `GetFunction` returns `None`.